src/components/StatisticsTile.js

```diff
@@ -19,7 +19,12 @@ class StatisticsTile extends Component {
   numTriggeredAlarms() {
     return this.props.alarms.data.filter(alarm => alarm.activeWarning()).length;
   }
-
+  bracketifyPluralization(title) {
```

Comment:
Aantal meegeven als argument

Reply:
Aangezien we nu hardoced _alarm/alarms_ willen laten zien:
```
  pluralizeAlarms(n) {
    return n === 1
      ? "alarm"
      : "alarms"
  }
```